pkg/asset/installconfig/aws/permissions.go

```diff
@@ -682,3 +696,20 @@ func includesZones(installConfig *types.InstallConfig) bool {
 
 	return len(mpool.Zones) > 0 || len(installConfig.AWS.Subnets) > 0
 }
+
+func includesWavelengthZones(installConfig *types.InstallConfig) bool {
+	isWLZoneRegex := regexp.MustCompile(`wl\d\-.*$`)
```

Comment:
nit: Can we add a note where the pattern comes from or what is expected?

Reply:
Done